app/frontend/translations.ts

```diff
@@ -130,6 +136,15 @@ const translations = {
   PoolRegNoTtl: () =>
     'TTL parameter is missing in the transaction. It is explicitly required even for the Allegra era.',
   PoolRegTxParserError: ({message}) => `Parser error: ${message}`,
+
+  Error: ({message}) => {
+    const errors = {
+      'NotFoundError: The device was disconnected.': `${message}${ledgerConnectError}`,
+      'AbortError: The transfer was cancelled.': `${message}${ledgerConnectError}`,
+      "SyntaxError: Failed to execute 'postMessage' on 'Window': Invalid target origin 'chrome-extension://kmendfapggjehodndflmmgagdbamhnfd' in a call to 'postMessage'": `${message}`,
+    }
+    return errors[message]
```

Comment:
this can get resolved to undefined if the message is lacking which is probably not desired, something like

```
return (message in errors) ? errors[message] ? message
```

Reply:
Actually, the undefined behavior is desired, we check whether an error has a translation and if not we display unexpectedErrorModal, so its crucial this returns undefined, otherwise all generic errors with the name `Error` would be "expected".

Reply:
I see - that's probably worth explicitly acknowledging with a comment so nobody is tempted to "fix" the function interface

Reply:
You are right, thanks for the suggestion, we should refactor this as well, but as mentioned in the description, this is rather a hotfix.

Reply:
hotfixes without appropriate TODO comments have high chances of becoming permanent fixes :D

Reply:
True.. :D